src/lib.rs

```diff
@@ -88,26 +87,23 @@ fn update(
     mut text_query: Query<&mut Text, With<ScreenDiagsText>>,
 ) {
     if let Some(mut state) = state_resource {
-        if state.update_now || state.timer.tick(time.delta()).just_finished() {
-            if state.timer.paused() {
-                // Time is paused so remove text
-                for mut text in text_query.iter_mut() {
-                    let value = &mut text.sections[0].value;
-                    value.clear();
-                }
-            } else {
-                let fps_diags = extract_fps(&diagnostics);
-
-                for mut text in text_query.iter_mut() {
-                    let value = &mut text.sections[0].value;
-                    value.clear();
-
-                    if let Some(fps) = fps_diags {
-                        write!(value, "{}{:.0}", STRING_FORMAT, fps).unwrap();
-                    } else {
-                        value.clear();
-                        write!(value, "{}", STRING_MISSING).unwrap();
-                    }
+        if state.timer.paused() {
+            // Timer is paused so remove text
+            for mut text in text_query.iter_mut() {
+                text.sections[0].value.clear();
```

Comment:
This clears the text every frame when the timer is paused. My code does this once only when disable is set.

Reply:
Shoot, you're right. Fixed by checking the update_now flag.